client/my-sites/comments/style.scss

```diff
@@ -75,6 +75,15 @@
 	}
 }
 
+.comment-list__header {
+	overflow: visible;
+
+	&.is-sticky .header-cake {
+		box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1), 0 0 56px rgba(0, 0, 0, 0.075);
+		margin: 9px 0;
```

Comment:
this margin leaves a gap between the sticky header and the Calypso header.

Reply:
+1 for removing the gap ^^